# image data is always 2bpp yuyv
# if other formats/compressessions are desired, more properties will be needed.
# images must be same size.
Header header
sensor_msgs/Image left_image
sensor_msgs/Image right_image
